locales/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "modules": "Course modules",
+  "modules-not-found": "No course modules found"
+}
```

Comment:
Add `files.insertFinalNewline": true` to your VS Code user settings :slightly_smiling_face: